controller/share_manager_controller.go

```diff
@@ -777,7 +779,15 @@ func (c *ShareManagerController) createPodManifest(sm *longhorn.ShareManager, an
 // this scenario would lead to the node no longer being able ot be drained
 // since the volume would never be unattached which prevents the engine from being removed
 func (c *ShareManagerController) shouldClaimOwnership(sm *longhorn.ShareManager) bool {
```

Comment:
You can rename this to isResponsibleFor same as the other controllers, since we are now doing the same more or less :)

Reply:
Thank you. Fixed